app/controllers/invitation_controller.rb

```diff
@@ -0,0 +1,22 @@
+class InvitationController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+    user = GithubService.new(current_user).get_user(params[:handle])
+    github_user = GithubUser.new(user)
+
+    if github_user.registered_user?
```

Comment:
@nagerz Does this look good? We already had this method. We're basically not allowing them to invite anyone already in the app, including themselves.

Reply:
Looks great